openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -114,14 +114,18 @@ <h3 translate>Election result</h3>
       </div>
       <div class="results">
         <div ng-repeat="option in poll.options">
-          <strong>{{ option.candidate.get_full_name() }}</strong>
-          <div ng-if="option.votes.length > 0">
-            <div ng-repeat="vote in option.votes">
-              {{ vote.value}}: {{ vote.weight}}
-            </div>
+          <div ng-if="poll.yesnoabstain && option.votes.length > 0">
+            <strong>{{ option.candidate.get_full_name() }}</strong><br>
+            <translate>Yes</translate>: {{ option.votes[0].weight }}<br>
```

Comment:
Ah this is tricky. We assume an array like this:
```json
[
  {
    "value": "Yes",
    "weight": "42"
  },
  {
    "value": "No",
    "weight": "-1"
  },
  {
    "value": "Abstain",
    "weight": "2"
  }
]
```
But it may also be:
```json
[
  {
    "value": "No",
    "weight": "-1"
  },
  {
    "value": "Yes",
    "weight": "42"
  },
  {
    "value": "Abstain",
    "weight": "2"
  }
]
```
Then the template output would be sliently wrong. At the moment the server always sends "Yes" as first element but this is not "protected" very well.

Maybe add a comment here?